### ISSUE TITLE
Jaws announcements not working - ARIA role should be Slider instead of Separator

### ISSUE THREAD

**Issue by sumegha26** [angular-split/angular-split#446]
**Describe the bug**
When Jaws is ON and I focus on gutter and press arrow keys - Left/Right or Up/Down gutter not getting resized it remains in same position

Suggestion -

Using Slider role on gutter instead of Separator works

Related issues - https://github.com/angular-split/angular-split/issues/349

**Comment by sumegha26** [comment]
any update on this ?

**Comment by Harpush** [comment]
Considering there was a PR to flip it to role separator from slider - we need to decide whether we want to allow it to be customizable or deicide which of the roles is the correct one for a split component

**Comment by SanderElias** [comment]
@Harpush is this something we still need to attend to? If not, we can close this issue.

**Comment by Harpush** [comment]
I am not sure... I am not that knowledgeable about ARIA and what's correct as role here

**Comment by SanderElias** [comment]
Euhm. I did dig in a bit. There is a catch-22 here.
The thing is, it is _both_ a `separator` (as discussed in #349), but at the same time it _is also_ a `slider` as it allows you to "input a new value for the placing of the separator"

I will try to figure out if adding _both_ is allowed.

**Comment by Harpush** [comment]
We can also add an input for it... Might be an overkill though
